Makefile

```diff
@@ -3,6 +3,7 @@ include versions.env
 # Common variables
 OUTDIR := out/
 
+CHECKCMDS = $(foreach cmd,$(1),$(if $(shell which $(cmd)),$(shell which $(cmd)),$(error "Command [$(cmd)] not available. Please install it first.")))
```

Comment:
Is the "foreach" here so this can be called with multiple params like ` $(call CHECKCMDS,cmd1,cmd2,..,cmdN)` ?

Reply:
Yes!